browser/ui/BUILD.gn

```diff
@@ -801,16 +803,22 @@ source_set("browser_tests") {
     ]
 
     deps = [
+      "//brave/app:command_ids",
       "//brave/browser/ui",
       "//brave/components/constants",
       "//chrome/browser",
       "//chrome/browser/devtools:test_support",
       "//chrome/browser/profiles:profile",
       "//chrome/browser/ui",
       "//chrome/test:test_support_ui",
+      "//chrome/test:unit_tests",
```

Comment:
with this line you effectively added browser_tests->unit_tests dependency.
you should definitely not do that. maybe you only need to depend on gtest?

Reply:
I picked wrong target for `chrome/browser/renderer_context_menu/render_view_context_menu_test_util.h` among below three candidates. `//chrome/test:test_support` is picked. Thanks!
```
ERROR at //brave/browser/ui/text_recognition_browsertest.cc:20:11: Include not allowed.
#include "chrome/browser/renderer_context_menu/render_view_context_menu_test_util.h"
          ^------------------------------------------------------------------------
It is not in any dependency of
  //brave/browser/ui:browser_tests
The include file is in the target(s):
  //chrome/test:interactive_ui_tests
  //chrome/test:test_support
  //chrome/test:unit_tests
at least one of which should somehow be reachable.
```